src/smartbanner.js

```diff
@@ -74,6 +74,7 @@ export default class SmartBanner {
     let parser = new OptionParser();
     this.options = parser.parse();
     this.platform = Detector.platform();
+    this.platformOS = this.platform && this.platform.replace(' tablet','').replace(' phone','');
```

Comment:
As by what was discussed, this could be best named differently, e.g. `device` that'd then receive fully-parsed value from detector.